bpfman/src/multiprog/tc.rs

```diff
@@ -32,8 +32,8 @@ use crate::{
     },
     utils::{
         bytes_to_string, bytes_to_u16, bytes_to_u32, bytes_to_u64, bytes_to_usize, enter_netns,
-        nsid, should_map_be_pinned, sled_get, sled_get_option, sled_insert,
-        tc_dispatcher_db_tree_name, tc_dispatcher_link_id_path, tc_dispatcher_rev_path,
+        should_map_be_pinned, sled_get, sled_get_option, sled_insert, tc_dispatcher_db_tree_name,
```

Comment:
did u intentionally deleted `nsid` here ?

Reply:
I did.  We were calling `nsid(netns.clone())?` from nsid below, but the nsid is already saved to the database, so I replaced that with `self.get_nsid()?` which doesn't require the caller to be in the netns to work.